email_templates/suppliers_new_briefs_questions_answers.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+</head>
+<body>
+Dear supplier,
+<br /><br />
+In the last day, buyers answered questions about:
```

Comment:
This is a content decision, not really decided by us, should we go to content with that?